src/app/routes/_testimonial.router.js

```diff
@@ -82,4 +99,33 @@ export default (app, router, io) => {
     }
   });
 
+
+
+  router.put('/api/testimonial', async (req, res) => {
```

Comment:
This route is used for updating the testimonial. Please create a separate one for toggling the review state.